frontend/src/app/[locale]/not-found.tsx

```diff
@@ -24,7 +24,7 @@ export default function NotFound() {
     <>
       <BetaAlert />
       <GridContainer className="padding-y-1 tablet:padding-y-3 desktop-lg:padding-y-15 measure-2">
-        <h1 className="nj-h1">{t("title")}</h1>
```

Comment:
looks like maybe a remnant from the nj ui project?

Reply:
Good catch.